src/effect.cpp

```diff
@@ -771,7 +771,7 @@ void effect::set_duration( const time_duration &dur, bool alert )
         set_intensity( duration / eff_type->int_dur_factor + 1, alert );
     }
 
-    add_msg( m_debug, "ID: %s, Duration %d", get_id().c_str(), to_turns<int>( duration ) );
+    add_msg( m_debug, "ID: %s, Duration %s", get_id().c_str(), to_string( duration ) );
```

Comment:
Further profiling showed that was not a good change - `to_string` calls translation routines which are pretty slow.

![image](https://user-images.githubusercontent.com/16213433/90447133-58cd3000-e0eb-11ea-949b-7f092310aa97.png)